src/cmd/manage_nsfs.js

```diff
@@ -382,7 +382,7 @@ async function fetch_account_data(argv, config_root, from_file) {
         const raw_data = (await nb_native().fs.readFile(fs_context, from_file)).data;
         data = JSON.parse(raw_data.toString());
     }
-    _validate_access_keys(argv);
+    if (action !== 'list' && action !== 'delete') _validate_access_keys(argv);
```

Comment:
I think there is an issue here - copying from slack - 
`remove access_key identifier makes sense from 'update' and 'delete' operations but they do want list/status by access_key`
please use the actions from ACTIONS array at the top of the file. (for example ACTIONS.LIST)